src/Tasks/DownloadFile.cs

```diff
@@ -60,6 +60,11 @@ public sealed class DownloadFile : TaskExtension, ICancelableTask
         [Required]
         public string SourceUrl { get; set; }
 
+        /// <summary>
+        /// Gets or sets the number of milliseconds to wait before the request times out.
+        /// </summary>
+        public int Timeout { get; set; } = 100_000;
```

Comment:
Team triage:
Thanks for this PR! We think it's best to start with no timeout by default and let users set it to a different value if they want. Otherwise looks good!

Reply:
100 seconds is exact default for the HttpClient, so existing behavior preserved. If remove default value, I have to add custom logic which map 0 => 100 seconds which probably not what's expected.